commerce/bookstore/python/load_data.py

```diff
@@ -0,0 +1,286 @@
+import csv, uuid, random
+from typedb.client import TypeDB, SessionType, TransactionType
+
+data_path = "data/"  # path to csv files to import/load data
+db = 'bookstore'  # Name of the DB to connect on the TypeDB
```

Comment:
```suggestion
db = "bookstore"  # Name of the DB to connect on the TypeDB
```
Let's use consistent quotes - I'd suggest speech marks everywhere, as this is what we use in our other Python projects.

Apply this change to all Python scripts.

Reply:
Fixed here (double quotes by default) https://github.com/vaticle/typedb-examples/pull/143/commits/6d50dec3400da9693ed442714a99918bf2080987